pkg/templates/kubechainsaw/template_test.go

```
@@ -0,0 +1,306 @@
+	tmpFile, err := os.CreateTemp("", "suppressions-*.yaml")
+	require.NoError(t, err)
+	defer os.Remove(tmpFile.Name())
+
+	suppressionYAML := `suppressions:
+- rule_id: KC-002
+  resource_name: dangerous-role
+  reason: "accepted risk for test"
+`
+	_, err = tmpFile.WriteString(suppressionYAML)
+	require.NoError(t, err)
+	tmpFile.Close()
```

golangci-lint (2.12.2)

[low] G104: Errors unhandled

(gosec)